ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.14f1
-m_EditorVersionWithRevision: 2020.3.14f1 (d0d1bb862f9d)
+m_EditorVersion: 2020.3.31f1
+m_EditorVersionWithRevision: 2020.3.31f1 (6b54b7616050)
```

Comment:
I don't mind upgrading the version but I don't usually commit this file myself as it means everyone else has to download the latest version.